pandas/io/tests/parser/common.py

```diff
@@ -243,6 +243,8 @@ def test_unnamed_columns(self):
                                        'Unnamed: 4'])
 
     def test_duplicate_columns(self):
+        # TODO: add test for condition 'mangle_dupe_cols=False'
+        # once it is actually supported (gh-12935)
```

Comment:
do we have any other tests that touch mangle_dupe_cols (I 'll suppose not), but if we do, move them near here.